packages/core/src/types.ts

```diff
@@ -68,13 +75,220 @@ export interface Config {
+     * this is that if you don't know who you are talking to, it is easy for an
+     * attacker to hijack your encrypted connection, rendering encryption pointless.
+     *
+     * TRUST_SYSTEM_CA_SIGNED_CERTIFICATES is the default choice. For NodeJS environments, this
+     * means that you trust whatever certificates are in the default trusted certificate
+     * store of the underlying system. For Browser environments, the trusted certificate
+     * store is usually managed by the browser. Refer to your system or browser documentation
+     * if you want to explicitly add a certificate as trusted.
+     *
+     * TRUST_CUSTOM_CA_SIGNED_CERTIFICATES is another option for trust verification -
+     * whenever we establish an encrypted connection, we ensure the host is using
+     * an encryption certificate that is in, or is signed by, a certificate given
+     * as trusted through configuration. This option is only available for NodeJS environments.
+     *
+     * TRUST_ALL_CERTIFICATES means that you trust everything without any verifications
+     * steps carried out.  This option is only available for NodeJS environments and should not
+     * be used on production systems.
+     *
+     * @type {'TRUST_SYSTEM_CA_SIGNED_CERTIFICATES'|'TRUST_CUSTOM_CA_SIGNED_CERTIFICATES'|'TRUST_ALL_CERTIFICATES'|undefined}
+     */
+    this.trust = undefined
+
+    /**
+     * List of one or more paths to trusted encryption certificates.
+     *
+     * This only works in the NodeJS bundle,
+     * and only matters if you use "TRUST_CUSTOM_CA_SIGNED_CERTIFICATES".
+     *
+     * The certificate files should be in regular X.509 PEM format.
+     *
+     * For instance, ['./trusted.pem']
+     *
+     * @type {?string[]}
+     * @see {@link Config#trust}
+     */
+    this.trustedCertificates = []
+
+    /**
+     * The maximum total number of connections allowed to be managed by the connection pool, per host.
+     *
+     * This includes both in-use and idle connections.
+     *
+     * **Default**: ```100```
+     *
+     * @type {number|undefined}
+     */
+    this.maxConnectionPoolSize = 100
+
+    /**
+     * The maximum allowed lifetime for a pooled connection in milliseconds.
+     *
+     * Pooled connections older than this
+     * threshold will be closed and removed from the pool. Such discarding happens during connection acquisition
+     * so that new session is never backed by an old connection. Setting this option to a low value will cause
+     * a high connection churn and might result in a performance hit. It is recommended to set maximum lifetime
+     * to a slightly smaller value than the one configured in network equipment (load balancer, proxy, firewall,
+     * etc. can also limit maximum connection lifetime). No maximum lifetime limit is imposed by default. Zero
+     * and negative values result in lifetime not being checked.
+     *
+     * **Default**: ```60 * 60 * 1000 // 1 hour```
```

Comment:
This is probably my ignorance, but how does the `//` get parsed? If it stays raw, then maybe `=` is more appropriate?

Reply:
I think it would look nicer, so that `// 1 minute` doesn't seem part of the default. Of course nobody will truly believe so, and anybody who believes so would be better off not writing software, but it still requires an extra second for appropriate parsing to the human eye that we could help avoid. It only makes sense if we change the format for all options though, not only for this one.

Reply:
I had addressed the comment and improve all the default values format.